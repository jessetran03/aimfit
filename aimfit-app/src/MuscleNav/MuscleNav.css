.muscle-group {
  padding: 20px;
}

.muscle-group li {
  display: inline;
}