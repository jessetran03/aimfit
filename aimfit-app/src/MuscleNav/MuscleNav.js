import React, { Component } from 'react'
import './MuscleNav.css'

export default class MuscleNav extends Component {
  render() {
    return (
      <>
        <section className="muscle-group">
          <h2>Muscle Group</h2>
          <ul>
            <li><button>Chest</button></li>
            <li><button>Back</button></li>
            <li><button>Quads</button></li>
            <li><button>Hamstrings</button></li>
            <li><button>Calves</button></li>
            <li><button>Triceps</button></li>
            <li><button>Biceps</button></li>
            <li><button>Shoulders</button></li>
            <li><button>Traps</button></li>
          </ul>
        </section>
      </>
    )
  }
}