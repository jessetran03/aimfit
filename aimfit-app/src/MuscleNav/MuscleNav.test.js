import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import MuscleNav from './MuscleNav'
import renderer from 'react-test-renderer'

it('renders MuscleNav without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(
    <BrowserRouter>
      <MuscleNav />
    </BrowserRouter>,
    div
  )
  ReactDOM.unmountComponentAtNode(div)
})

it('render MuscleNav without data', () => {
  const wrapper = renderer.create(
    <BrowserRouter>
      <MuscleNav />
    </BrowserRouter>
  )
    .toJSON();
    expect(wrapper).toMatchSnapshot()
  })