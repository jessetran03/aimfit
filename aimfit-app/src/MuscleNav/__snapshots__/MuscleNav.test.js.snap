// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render MuscleNav without data 1`] = `
<section
  className="muscle-group"
>
  <h2>
    Muscle Group
  </h2>
  <ul>
    <li>
      <button>
        Chest
      </button>
    </li>
    <li>
      <button>
        Back
      </button>
    </li>
    <li>
      <button>
        Quads
      </button>
    </li>
    <li>
      <button>
        Hamstrings
      </button>
    </li>
    <li>
      <button>
        Calves
      </button>
    </li>
    <li>
      <button>
        Triceps
      </button>
    </li>
    <li>
      <button>
        Biceps
      </button>
    </li>
    <li>
      <button>
        Shoulders
      </button>
    </li>
    <li>
      <button>
        Traps
      </button>
    </li>
  </ul>
</section>
`;
